test/jdk/sun/management/jmxremote/startstop/JMXStatusTest.java

```diff
@@ -33,7 +33,7 @@
 
 /**
  * @test
- * @bug 8023093 8138748 8142398
+ * @bug 8023093 8138748 8142398 8303102
```

Comment:
As this test doesn't explicitly test/check the issue in 8303102 I don't think it needs to belisted on the @bug line. (And if it does then the copyright year needs t be updated too).

Reply:
I have found this issue by using this test on a host with long hostname, it does validate the output of the command and fails if it truncated.